File: app/src/main/java/com/example/purepawapp/ui/order/OrderSuccessFragment.java
package com.example.purepawapp.ui.order;

import android.os.Bundle;
import android.view.View;

import androidx.navigation.fragment.NavHostFragment;

import com.example.purepawapp.databinding.FragmentOrderSuccessBinding;
import com.example.purepawapp.ui.common.BaseFragment;

public class OrderSuccessFragment extends BaseFragment<FragmentOrderSuccessBinding> {

    public OrderSuccessFragment() {
        super(FragmentOrderSuccessBinding::inflate);
    }

    @Override
    public void onViewCreated(View view, Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        getBinding().btnDone.setOnClickListener(v -> NavHostFragment.findNavController(this).popBackStack());
    }
}

File: app/src/main/java/com/example/purepawapp/ui/order/OrderSuccessFragment.kt
package com.example.purepawapp.ui.order

import android.os.Bundle
import android.view.View
import androidx.navigation.fragment.findNavController
import com.example.purepawapp.databinding.FragmentOrderSuccessBinding
import com.example.purepawapp.ui.common.BaseFragment

class OrderSuccessFragment : BaseFragment<FragmentOrderSuccessBinding>(FragmentOrderSuccessBinding::inflate) {

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        binding.btnDone.setOnClickListener {
            findNavController().popBackStack()
        }
    }
}
